.ci_support/migrations/petsc313.yaml

```diff
@@ -0,0 +1,7 @@
+__migrator:
```

Comment:
@jan-janssen Could you explain what this is doing/what migration is as I'm unfamiliar with these types of files?

Reply:
This is a standard migrator file for the autotick bot. This PR is done correctly. Additionally, the changes in the pinnings that make it pass are what is stopping the bot. We should merge this one.

Reply:
Sounds good.